utils/wasm-builder/src/wasm_project.rs

```diff
@@ -354,10 +356,20 @@ pub const WASM_EXPORTS: &[&str] = &{:?};
             .context("unable to write `wasm_binary.rs`")?;
         }
 
-        Ok(())
+        self.force_rerun_on_next_run(&from_path)
```

Comment:
much more common pattern
```suggestion
        let _ = self.force_rerun_on_next_run(&from_path)?;
        
        Ok(())
```

Reply:
Maybe, it is a more common pattern, but definitely not for clippy. It does not like it ;)

Reply:
You might remove `let _`, but I vote for your code as it is shorter.

```rust
// Just in case for clippy. But expression-based code is preferable to me.
self.force_rerun_on_next_run(&from_path)?;
Ok(())
```